src/yeahml/build/components/util.py

```diff
@@ -1,19 +0,0 @@
-import functools
```

Comment:
possibly [mark for keep] -- we have to keep either this or the configure file (above note)